src/tests/ftest/util/soak_test_base.py

```diff
@@ -144,7 +148,8 @@ def pre_tear_down(self):
         if self.all_failed_jobs:
             errors.append("SOAK FAILED: The following jobs failed {} ".format(
                 " ,".join(str(j_id) for j_id in self.all_failed_jobs)))
-
+        # cleanup any remaining jobs
+        job_cleanup(self.log, self.hostlist_clients)
```

Comment:
In the future it would be nice to remove the pre_tear_down method now that we can use register_cleanup to more effectively handle tearDown operations. Just a note; not a requested change.